apps/researcher/src/app/[locale]/object/[id]/page.tsx

```diff
@@ -88,6 +88,13 @@ export default async function Details({params}: Props) {
     return <div data-testid="no-object">{t('noObject')}</div>;
   }
 
+  const imagesWithAlt =
+    object.images?.map((image, i) => ({
+      ...image,
```

Comment:
This merges all properties of `image` (e.g. `id` and `contentUrl`). But according to the `Props` interface of `gallery.tsx` it should only contain `id`. Perhaps selectively merge properties?